protos/gossip/message.proto

```diff
@@ -130,6 +130,12 @@ message GossipMessage {
         // Encapsulates private data used to distribute
         // private rwset after the endorsement
         PrivateDataMessage private_data = 25;
+
+        // Contains validation results
+        ValidationResultsMessage validation_results_msg = 50;
```

Comment:
i wonder if we should leave a larger gap.

Reply:
We will increase it in the next change

Reply:
I remove it